client/my-sites/marketplace/components/reviews-cards/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import {
+	useMarketplaceReviewsQuery,
+	ProductProps,
+} from 'calypso/data/marketplace/use-marketplace-reviews';
+import { getCurrentUserId } from 'calypso/state/current-user/selectors';
+import { IAppState } from 'calypso/state/types';
+import { MarketplaceReviewCard } from './review-card';
+import './style.scss';
+
+export const MarketplaceReviewsCards = ( props: ProductProps ) => {
+	const translate = useTranslate();
+	const currentUserId = useSelector( ( state: IAppState ) => getCurrentUserId( state ) );
+	const { data: reviews } = useMarketplaceReviewsQuery( { ...props, perPage: 2, page: 1 } );
+
+	if ( ! isEnabled( 'marketplace-reviews-show' ) ) {
+		return null;
+	}
+
+	if ( ! Array.isArray( reviews ) || ! reviews || 'message' in reviews ) {
```

Comment:
This check `Array.isArray(reviews)` is great here as it avoids checking it again further below the code. Could we then remove the checks in lines 29 and 58 for decluttering the code?

Reply:
That was actually the idea, but then I forgot it :) 

Thanks for pointing that https://github.com/Automattic/wp-calypso/pull/85323/commits/d827fc6d4a9e264d833f68c74f6b39dc49a67489